### PR TITLE
Fix cookie window issues

### PR DESCRIPTION
**Pull request by kaushiksriram16**
* Delayed appearance of cookie window using setTimeout
* Removed all the buttons of cookie window header
* Removed the disagree button on the cookie window

   cw

### CODE REVIEW

**Comment by arpansac** [comment]
@kaushiksriram16 can you please fix the bug on line 63 in the bug tracker as well in this same PR

**Comment by kaushiksriram16** [comment]
> @kaushiksriram16 can you please fix the bug on line 63 in the bug tracker as well in this same PR

I made it to appear 2-3 seconds after the page loads but just now I realized that it is also mentioned that it should appear only if it's browser, So I have to work on that.

File: projects/commudle-admin/src/app/app.component.scss [hunk 1]
```diff
@@ -339,3 +339,13 @@
     }
   }
 }
+
+:host::ng-deep {
+  .cdk-overlay-container {
+    nb-card-header {
+      button {
+        display: none;
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/app.component.scss]: This is affecting the join community dialogue box and follow expert dialogue box on the home page.
So, as much as possible, let's avoid the `:host` selector and use classes to make the required changes

**Review by Ashwin-op — CHANGES_REQUESTED** [review]
Please update the PR with a pull from the development branch

**Pull request merged**